base_classes/NXentry.nxdl.xml

```diff
@@ -51,12 +51,34 @@
 		(**required**) :ref:`NXentry` describes the measurement. 
 		
 		The top-level NeXus group which contains all the data and associated
-		information that comprise a single measurement. It is mandatory that there is at least one
-		group of this type in the NeXus file.
-	</doc>
+		information that comprise a single measurement. 
+		It is mandatory that there is at least one
+		group of this type in the NeXus file.	</doc>
 	
-	<group type="NXdata" minOccurs="1">
-		<doc>The required data group</doc>
+	<group type="NXdata">
+		<doc>
+			The data group
+		
+			.. note:: Before the NIAC2016 meeting [#], at least one
+			   :ref:`NXdata` group was required in each :ref:`NXentry`` group.
+			   At the NIAC2016 meeting, it was decided to make :ref:`NXdata` 
+			   an optional group in :ref:`NXentry`` groups for data files that 
+			   do not use an application definition.
```

Comment:
It is recommended strongly that a NXdata group be provided in all NeXus data files and only omitted when such data is not practical or possible.